control-plane/grpc/src/operations/pool/traits.rs

```diff
@@ -385,3 +387,43 @@ impl From<PoolSpecStatus> for common::SpecStatus {
         }
     }
 }
+
+// Create pool creation event message from pool data.
+impl Event<Pool> for EventMessage {
+    fn get_event(value: &Pool) -> String {
+        let event_meta = EventMeta::new();
+        let event = EventMessage {
+            category: Category::Pool as i32,
+            action: Action::CreateEvent as i32,
+            target: value.id().to_string(),
+            metadata: Some(EventMeta {
+                source: Some(EventSource {
+                    component: "core-agent".to_string(),
```

Comment:
this is not correct, this is a transport crate, any component could be using these. Maybe should be set by the caller?